WORKSPACE

```diff
@@ -277,8 +277,8 @@ http_file(
 http_file(
     name = "busybox_ppc64le",
     executable = True,
-    sha256 = "7f896a2560ef3e2d11438018fef2fbdf8c2a6a2e4b84b1bcbaacc669967532d4",
-    urls = ["https://busybox.net/downloads/binaries/1.31.0-defconfig-multiarch-musl/busybox-powerpc64"],
+    sha256 = "469297ea9293df0dcb6c3f8d344eaf9f9b6ec1732696ffe86994f87c3600423b",
+    urls = ["https://github.com/docker-library/busybox/raw/c0125333c4c3dfa4b9e5fd9fe6fbb875242f3613/stable/musl/busybox.tar.xz"],
```

Comment:
I can't tell where this is coming from, I don't see the files checked in here: https://github.com/docker-library/busybox/tree/master/stable/musl

Reply:
its in `dist-ppc64le` branch - https://github.com/docker-library/busybox/tree/dist-ppc64le/stable/musl

Reply:
Ah thanks! Could you leave a comment here on that with some instructions on how to update it?

Reply:
The `ppc64le` busybox tarball is present at [stable/musl/busybox.tar.xz](https://github.com/docker-library/busybox/blob/dist-ppc64le/stable/musl/busybox.tar.xz) in [dist-ppc64le](https://github.com/docker-library/busybox/tree/dist-ppc64le) branch of [docker-library/busybox](https://github.com/docker-library/busybox) repo. However, these files are frequently update hence directly using `https://github.com/docker-library/busybox/raw/dist-ppc64le/stable/musl/busybox.tar.xz` link in  `WORKSPACE` will result into failure due to `sha256` mismatch. To avoid this, we can stick to a particular `commit` and update it with `latest` whenever required.

To update:
- Get the latest `commit hash` from [dist-ppc64le](https://github.com/docker-library/busybox/commits/dist-ppc64le/stable/musl/busybox.tar.xz) branch.
- Substitute it in the link: `https://github.com/docker-library/busybox/raw/<latest-commit-hash>/stable/musl/busybox.tar.xz`
- Update the sha256 value. Since [github api](https://api.github.com/repos/docker-library/busybox/contents/stable/musl/busybox.tar.xz?ref=c0125333c4c3dfa4b9e5fd9fe6fbb875242f3613) doesn't give `sha256` value, it can be obtained  using `sha256sum` command.

Reply:
That comment looks great! Sorry for not being more clear, could you leave that in a comment in the code somewhere for whoever comes along to update it later?

Reply:
Added a comment in `WORKSPACE` file. @dlorenc